components/playback/controls/player/PopInput.vue

```diff
@@ -0,0 +1,231 @@
+    input_width: { type: Number, default: 0 }, // textbox_width (int)  [pixels]
+    block: { type: Boolean, default: false, required: false }, // disabled (optional bool=False) (not able to type into input)
+  },
+  data() {
+    return {
+      input_value_key: this.user_key,
+      input_width_background: this.input_width + 4, // This is required as the red/green boxes around the input widget requirement based on feedback introduced the need its not in Mockflow
+      // very essential else the input box would appear poping out on the right side outside the background, request to consult Eli or Raghu
+    };
+  },
+  computed: {
+    input_key_action: function () {
+      // This is a very sensitive computed function as its invoked on every key entry by user action
+      // the function would never have any processing its only responsible to pass the value of string to the parent component
+      // any modification to add logic might impact depedent functionalities, request to consult Eli or Raghu
+      this.$emit("update:user_key", this.input_value_key);
+      return this.invalid_text === "";
+    },
+    input_height_background: function () {
+      return this.title_label !== "" ? 100 : 60;
+    },
+    input_widget_top: function () {
+      return this.title_label !== "" ? 40 : 0;
+    },
+    input_feedback_top: function () {
+      return this.title_label !== "" ? 88 : 48;
+    },
+  },
+  methods: {},
+};
+</script>
+<style type="text/css">
+.div__popinput-background {
+  transform: rotate(0deg);
+  box-sizing: border-box;
+  padding: 0px;
+  margin: 0px;
+  background: rgb(17, 17, 17);
+  position: absolute;
+  /* height: 100px; */
+  top: 0px;
+  left: 0px;
+  visibility: visible;
+  border: 2px solid rgb(0, 0, 0);
+  border-radius: 0px;
+  box-shadow: none;
+  z-index: 3;
+  pointer-events: all;
+}
+
+.span__popinput-content-label {
+  pointer-events: all;
+  align: center;
+  line-height: 100%;
+  transform: rotate(0deg);
+  overflow: hidden;
+  position: absolute;
+  height: 30px;
+  top: 0px;
+  left: 5px;
+  padding: 5px;
+  visibility: visible;
+  user-select: none;
+  font-family: Muli;
+  font-weight: normal;
+  font-style: normal;
+  text-decoration: none;
+  font-size: 17px;
+  color: rgb(183, 183, 183);
+  text-align: center;
+  z-index: 25;
+}
+
+.span__popinput-controls-content-input-txt-widget {
+  padding-left: 0px;
+  padding-right: 0px;
+  overflow: hidden;
+  white-space: nowrap;
+  text-align: left;
+  font-weight: normal;
+  transform: translateZ(0px);
+  position: absolute;
+  height: 45px;
+  line-height: 45px;
+  top: 0px;
+  left: 0px;
+  user-select: none;
+  font-family: "Anonymous Pro";
+  font-style: normal;
+  text-decoration: none;
+  font-size: 17px;
+  color: rgb(255, 255, 255);
+  background-color: #2f2f2f;
+}
+
+.div__popinput-controls-content-input-widget {
+  pointer-events: all;
+  transform: rotate(0deg);
+  overflow: hidden;
+  position: absolute;
+  height: 45px;
+  /* top: 40px; */
```

Comment:
remove old commented code when submitting pull requests, unless there is a specific reason to leave it in